setup.cfg

```diff
@@ -30,15 +31,21 @@ classifiers =
 ignore=W503
 max-line-length=160
 
-[entry_points]
-console_scripts =
-    ansible-builder = ansible_builder.cli:run
+[options]
+include_package_data = true
+install_requires =
+    PyYAML
+    requirements_parser
+    bindep
+    jsonschema
+
+[options.package_data]
+ansible_builder.target_scripts =
+    *
 
-[files]
-packages =
-    ansible_builder
+python_requires = >=3.9
```

Comment:
This line needs to move up under `[options]`

Reply:
nice catch- that would've sucked...